pilot/cmd/pilot-discovery/main.go

```diff
@@ -162,8 +162,8 @@ func init() {
 		"Injection and validation service HTTPS address")
 	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.GrpcAddr, "grpcAddr", ":15010",
 		"Discovery service grpc address")
-	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.SecureGrpcAddr, "secureGrpcAddr", ":15012",
-		"Discovery service grpc address, with https")
+	discoveryCmd.PersistentFlags().StringVar(&serverArgs.DiscoveryOptions.SecureGrpcAddr, "secureGrpcAddr", ":15011",
+		"Discovery service grpc address, with https and spiffe certificates.")
```

Comment:
so this requires istio-pilot service?  istiod service doesn't have this port.

Reply:
This port will not be used anymore, its only to support 1.4 upgrade. In master we can probably remove it entirely

Reply:
Yes, that's the old 15011 which would be used by older sidecars ( not yet injected with 1.5 ).